src/Spout/Writer/XLSX/Helper/StyleHelper.php

```diff
@@ -139,7 +148,7 @@ protected function getCellXfsSectionContent()
         $content = '<cellXfs count="' . count($registeredStyles) . '">';
 
         foreach ($registeredStyles as $style) {
-            $content .= '<xf numFmtId="0" fontId="' . $style->getId() . '" fillId="0" borderId="0" xfId="0"';
+            $content .= '<xf numFmtId="0" fontId="' . $style->getId() . '" fillId="' . $style->getId() .  '" borderId="0" xfId="0"';
```

Comment:
If I understand correctly, there will be as many fills declaration as there are styles. So even if none of the styles uses a bg color, we'll still define N fills with `patternType="none"` and use them here. Is that correct?
If my understanding is correct, any way this can optimized by having only one "no fill" declaration?